### DISCUSSION TITLE
Send Test smtp mail from site settings

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1209] [Ideas] [answered]
after configuring a smtp server it would be ideal to be able to send a test mail to verify that the configuration is correct.

**Comment by hishamco** [comment]
Make sense

**Comment by sbwalker** [comment, marked as answer]
PR #1213 adds this capability. Please note that SMTP is handled ansynchronusly in Oqtane so testing the SMTP configuration actually involves creating a Notification and allowing the Notification Job to attempt to deliver it.
